packages/slate-editor/src/extensions/placeholders/elements/StoryEmbedPlaceholderElement.stories.tsx

```diff
@@ -0,0 +1,270 @@
+    type: PlaceholderNode.Type.STORY_EMBED,
+    uuid: 'e57a4e5c-7769-4cbd-a159-a68be9373d26',
+    children: [{ text: '' }],
+};
+
+const attributes: RenderElementProps['attributes'] = {
+    'data-slate-node': 'element',
+    'data-slate-void': true,
+    ref: () => null,
+};
+
+let lastStoryId = 0;
+let lastNewsroomId = 0;
+
+const EN: CultureRef = {
+    name: 'English',
+    code: 'en',
+    locale: 'en',
+    direction: Culture.TextDirection.LTR,
+    native_name: 'English',
+    language_code: 'EN',
+};
+
+const THUMBNAIL = [
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-White.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Circus.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Mosque.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-FairyLights.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Goemetry.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Comb.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Floral.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Stripes.jpg',
+    'https://img.dummy-image-generator.com/abstract/dummy-48x48-Map.jpg',
+];
+
+const NEWSROOM = [
+    { name: 'Apple' },
+    { name: 'Banana' },
+    { name: 'Cinnamon' },
+    { name: 'Durian' },
+    { name: 'Elderberry' },
+];
+
+function newsroom(): NewsroomRef {
+    const id = ++lastNewsroomId;
+    return {
+        uuid: uuid.v4(),
+        /**
+         * @deprecated Please use `uuid` instead.
+         * @see uuid
+         */
+        id,
+        display_name: NEWSROOM[id % NEWSROOM.length].name,
+        thumbnail_url: THUMBNAIL[(id + 5) % THUMBNAIL.length],
+        name: NEWSROOM[id % NEWSROOM.length].name,
+        subdomain: NEWSROOM[id % NEWSROOM.length].name.toLowerCase(),
+        status: Newsroom.Status.ACTIVE,
+        is_active: true,
+        is_archived: false,
+        is_multilingual: true,
+        is_indexable: true,
+        timezone: 'Europe/Brussels',
+        time_format: 'HH:mm',
+        date_format: 'DD.MM.YYYY',
+        is_offline: false,
+        is_online: true,
+        url: NEWSROOM[id % NEWSROOM.length].name.toLowerCase() + '.prezly.com',
+        /**
+         * @deprecated Do not rely on these.
+         */
+        links: {
+            media_gallery_api: '',
+            analytics_and_visibility_settings: '',
+            categories_management: '',
+            company_info_settings: '',
+            contacts_management: '',
+            domain_settings: '',
+            edit: '',
+            gallery_management: '',
+            hub_settings: '',
+            languages: '',
+            look_and_feel_settings: '',
+            manual_subscription_management: '',
+            privacy_settings: '',
+            widget_settings: '',
+        },
+    };
+}
+
+function story(props: Partial<StoryRef> & Pick<StoryRef, 'uuid' | 'title'>): StoryRef {
```

Comment:
`story()` and `newsroom()` are both identical in `StoryEmbedPlaceholderElement` and `StoryBookarmarkPlaceholderElement`. Wouldn't it be better to move them to some kind of utils than can be used in both? something like `factory`?